app/src/main/java/com/ataulm/whatsnext/WhatsNextRepository.kt

```diff
@@ -59,13 +63,8 @@ internal class WhatsNextRepository(
                         rating = rating.toApiValue()
                 )
         )
-        // TODO: this is wasteful - the filmSummary won't have changed.
-        val apiFilm = letterboxdApi.film(letterboxdId)
         val apiFilmRelationship = apiFilmRelationshipUpdateResponse.data
-        Film(
-                filmSummaryConverter.convert(apiFilm),
-                filmRelationshipConverter.convert(apiFilmRelationship)
-        )
+        filmRelationshipConverter.convert(apiFilmRelationship)
```

Comment:
when we're updating a film relationship (whether user likes it/watched it etc), the summary of the film doesn't change - there's no need to load it again.